build.gradle

```diff
@@ -339,7 +357,6 @@ task compressTiny(dependsOn: ["tinyJar", "mergeTiny"], type: FileOutput){
 		fileInputStream.close()
 		outputStream.finish()
 		outputStream.close()
-		inputFile.delete()
```

Comment:
Ill need to double check this wasnt important

Reply:
The task deleted this file; I retain the file so that secondary enum field filling could use it.

Reply:
ah ok, thats fine.